policy/main/.manifest

```diff
@@ -0,0 +1,4 @@
+{
+    "revision": "v1.0.0",
```

Comment:
This doesn't need fixing just yet as the container image contains the git sha but I created a PR for a temporary solution to having this also set in the policy manifest as I think we will forget to update this and it may catch us out if we distribute policies by alternative methods -  https://github.com/mertbugrabicak/mequal/pull/2 but feel free to ignore

Reply:
Oh yes, this was something I wanted to do but forgot! We can definitely inject it during image build.

For the mequal bundle versioning, we can use our git sha, and for the external policies we define in our config json, I can create a revision field where we can concretely define what sha we are fetching the external policy from, and then inject those shas to their respective manifests.

Reply:
Will take in your changes and work from there

Reply:
Merged your changes, will now expand it to include external policies